croncert-config.yml

```diff
@@ -1738,6 +1738,111 @@ scrapers:
         date_location: "America/Los_Angeles"
         date_language: "en_US"
 
+  ##########
+  # Vevey
+  ##########
+  - name: CafeLitteraire
+    url: https://lecafelitteraire.ch/culture/
+    item: body.culture > div.site > main.site-main > article.is_page.culture.is_parent > div.entry-content > div.wp-block-getwid-section > div.wp-block-getwid-section__wrapper > div.wp-block-getwid-section__inner-wrapper > div.wp-block-getwid-section__content > div.wp-block-getwid-section__inner-content > ul.list.evenements > li
+    fields:
+      - name: "location"
+        value: "Café Litteraire"
+        type: text
+      - name: "city"
+        type: text
+        value: "Vevey"
+      - name: "type"
+        type: text
+        value: "concert"
+      - name: "sourceUrl"
+        type: url
+        value: "https://lecafelitteraire.ch/culture/"
+      - name: date
+        type: date
+        components:
+          - covers:
+              day: true
+              month: true
+            location:
+              selector: p.event_date > strong.has-green-color
+            layout: ["Monday 2 January"]
+          - covers:
+              time: true
+            location:
+              selector: p.event_date > span
+              regex_extract:
```

Comment:
I added the regex, so the scraper only takes the start time of the event. Before it'd just use the end time

Reply:
I only learned about the regexp part more recently